docs/FOR VALIDATORS/Join a Testnet.md

```diff
@@ -0,0 +1,158 @@
+# Join a Testnet
+
+## Pick a Testnet
+You specify the network you want to join by setting the genesis file and seeds. If you need more information about past networks, check our [testnets repo](https://github.com/imversed/testnets).
+
+| Testnet Chain ID | Description | Site | Version | Status |
+| --- | --- | --- | --- | --- |
+| `imversed_5555558-1` | imversed_5555558-1 Testnet | imversed_5555558-1 | v3.11 | `Live` |
+
+
+## Install `imversed`
+Follow the [installation](https://docs.imversed.com/validators/quickstart/installation.html) document to install the Imversed binary `imversed`.
+
+> Make sure you have the right version of imversed installed.
+
+### Save Chain ID
+We recommend saving the testnet `chain-id` into your `imversed`'s `client.toml`. This will make it so you do not have to manually pass in the chain-id flag for every CLI command.
+
+> See the Official [Chain IDs](https://docs.imversed.com/users/technical_concepts/chain_id.html#official-chain-ids) for reference
+
+```text
+imversed config chain-id imversed_5555558-1
+```
+
+## Initialize Node
+
+We need to initialize the node to create all the necessary validator and node configuration files:
+
+```text
+imversed init <your_custom_moniker> --chain-id imversed_5555558-1
+```
+
+> Monikers can contain only ASCII characters. Using Unicode characters will render your node unreachable.
+
+By default, the `init` command creates your `~/.imversed` (i.e `$HOME`) directory with subfolders `config/` and `data/`. In the `config` directory, the most important files for configuration are `app.toml` and `config.toml`.
+
+## Genesis & Seeds
+
+### Copy the Genesis File
+
+Check the `genesis.json` file from the [`archive`](https://archive.imversed.dev/imversed_5555558-1/genesis.json) and copy it over to the `config` directory: `~/.imversed/config/genesis.json`. This is a genesis file with the chain-id and genesis accounts balances.
+
+```text
+sudo apt install -y unzip wget
+wget -P ~/.imversed/config https://archive.imversed.dev/imversed_5555558-1/genesis.json
+```

```

Comment:
Change wget -P ~/.imversed/config https://archive.imversed.dev/imversed_5555558-1/genesis.json to https://storage.googleapis.com/static.fdvr.co/imversed/testnet/genesis.json

Reply:
Updated...

Reply:
Still link (https://archive.imversed.dev/imversed_5555558-1/genesis.json) instead of link (https://storage.googleapis.com/static.fdvr.co/imversed/testnet/genesis.json)